src/routes/indexable-indexation-route.php

```diff
@@ -177,17 +170,17 @@ public function __construct(
 		Indexable_Prepare_Indexation_Action $prepare_indexation_action,
 		Options_Helper $options_helper
 	) {
+		Abstract_Indexation_Route::__construct( $options_helper );
```

Comment:
This seems unusual to me, just checking: was this change intended?

Reply:
Yes and no 😅.
It should be:
```php
parent::__construct( $options_helper );
```
Since the constructor of the parent class changed (it now accepts an `Options_Helper`) any subclasses should call its parent's constructor with this new argument.

I also found a class (`Link_Indexing_Route`) which also inherits the `Abstract_Indexing_Route`, which did not have the call to the parent constructor, so I added it as well.

Reply:
Nice additional catch from your side!